.github/workflows/e2e.istio.yaml

```diff
@@ -129,3 +129,93 @@ jobs:
       - name: Clean up
         if: ${{ always() }}
         run: minikube delete
+
+  metrics-service:
+    runs-on: ubuntu-16.04
+    timeout-minutes: 60
+    name: MetricsService
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
+
+      - name: Build Docker Image
+        run: make docker
+
+      - name: Prepare envrionment
+        run: bash ${SCRIPTS_DIR}/pre.sh
+
+      - name: Install Minikube
+        run: bash ${SCRIPTS_DIR}/minikube.sh start
+
+      - name: Install Istio
+        run: bash ${SCRIPTS_DIR}/istio.sh --set profile=demo --set meshConfig.defaultConfig.envoyMetricsService.address=skywalking-oap.istio-system:11800
```

Comment:
I assume meta-exchange is default ON by today's Istio, right? I think we need to disable it manually?

Reply:
And e2e fails, please recheck.